cardinal/ecs/storage/engine.go

```diff
@@ -100,3 +100,7 @@ type NonceStorage interface {
 	GetNonce(key string) (uint64, error)
 	SetNonce(key string, nonce uint64) error
 }
+
+type IO interface {
+	Close() error
+}
```

Comment:
we can just use the interface defined in Go's `io` package: `io.Closer`, i believe it is more standard to do so

Reply:
Did not know this already existed.